### PR TITLE
Update manifest

### PR DESCRIPTION
**Pull request by masci**
### What does this PR do?

Updates manifest to latest version

### Motivation

Having the same as integrations-core

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [x] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: snmpwalk/manifest.json [hunk 1]
```diff
@@ -1,18 +1,15 @@
 {
   "maintainer": "help@datadoghq.com",
-  "manifest_version": "0.1.0",
-  "max_agent_version": "6.0.0",
-  "min_agent_version": "5.6.3",
+  "manifest_version": "1.0.0",
   "name": "snmpwalk",
+  "display_name": "Snmpwalk",
```
Comment (gzussa) [inline snmpwalk/manifest.json]: Is this the correct case?

File: snmpwalk/manifest.json [hunk 2]
```diff
@@ -1,18 +1,15 @@
 {
   "maintainer": "help@datadoghq.com",
-  "manifest_version": "0.1.0",
-  "max_agent_version": "6.0.0",
-  "min_agent_version": "5.6.3",
+  "manifest_version": "1.0.0",
```
Comment (gzussa) [inline snmpwalk/manifest.json]: why does the version here need to change?

Comment (masci) [reply snmpwalk/manifest.json]: number and type of fields changed in `1.0.0`

File: snmpwalk/manifest.json (hunk 1 above)
Comment (masci) [reply snmpwalk/manifest.json]: dunno, utility name is `snmpwalk` all lowercase. We capitalize the name elsewhere, see https://github.com/DataDog/integrations-core/blob/master/nfsstat/manifest.json#L6

**Pull request merged**